spec.bs

```diff
@@ -3587,6 +3625,25 @@ An auction config is a [=struct=] with the following items:
   Restricts the `generateBid()` script's runtime for all buyers without a timeout specified in
   [=auction config/per buyer timeouts=]. If the timeout expires, only the bid submitted via
   `setBid()` is considered.
+: <dfn>per buyer cumulative timeouts</dfn>
+:: Null, a {{Promise}}, failure, or an [=ordered map=] whose [=map/keys=] are [=origins=] and
+  whose [=map/values=] are [=durations=] in milliseconds.
+  [=map/Keys=] are buyers and must be valid HTTPS origins. [=map/Values=] are collective timeouts
+  for all interest groups of the buyer represented by the [=map/key=]. Includes the time of loading
+  scripts and signals, and running the `generateBid()` functions. Once the timer expires, the
+  affected buyer's interest groups may no longer generate any bids. All bids generated before the
+  timeout will continue to participate in the auction.
+
+  Note: Implementations <code><span class="allow-2119">should</span></code>

```

Comment:
Honestly this seems like a normative statement, so the `allow-2119` exception here seems a little out of place. Maybe just not make this a Note, and instead just make it a regular normative paragraph?

Reply:
This still seems like a Note, but this comment was marked as resolved?

Reply:
Hmm weird. I remember I made the change very clearly (e.g., rephrased the last sentence to use [=wait until configuration input promises resolve=]), maybe I stashed that change for some reason and forgot to get it back.
Anyways, resolved this time, and thanks for catching it.